float32 accel_opening 
float32 max_velocity 
float64 steer_angle 
float64 vehicle_velocity 
int8 direction_switch 
int8 light_signal 
int8 left_winker
int8 right_winker
int8 hone_signal